src/redux/slices/dive-logs/index.ts

```diff
@@ -101,6 +103,7 @@ export const diveLogsSlice = createSlice({
         state.error.status = false;
         state.error.message = null;
         state.diveLogs = normalizeData(action.payload.reviews);
+        state.orderedDiveLogs = action.payload.reviews;
```

Comment:
I'm not sure how this orders the dive log. The normalizeData function merely stores the dive logs in a way that operating on the items in state would be a very fast operation. It should have no effect on the ordering of the data.

Reply:
I've taken a look at it, and I can see what happened. The normalizing reversed the order.

Reply:
i think that it basically makes the order non-deterministic, and in general i think ends up sorting it by the ID. because it hashes the data by ID, and then re-constructs it as an array, so the order becomes whatever the implementation of constructing an array from dictionary keys is

Reply:
Did a minimal reproduction and it actually reorders when hashing as an object by id. So when reconstructing as an array, it is already sorted while in object shape.

Reply:
Also, this reordering only happens when the id keys are in forms where sorting is instinctive. For example, alphanumeric ids will not run into the same problem.